manifests/init.pp

```diff
@@ -31,6 +31,7 @@
   },
   $createkeys_path     = '/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin',
   $manage_eyaml_config = true,
+  $manage_keys         = true,
```

Comment:
of we can just enforce it to be a bool right? I rather prefer people using the right values than giving them options to use strings